dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/metadata/DefaultAnalyticalObjectImportHandler.java

```diff
@@ -84,7 +84,6 @@ private void handleDataDimensionItems( Session session, Schema schema, BaseAnaly
             if ( dataDimensionItem.getDataElementOperand() != null )
             {
                 preheatService.connectReferences( dataDimensionItem.getDataElementOperand(), bundle.getPreheat(), bundle.getPreheatIdentifier() );
```

Comment:
@vietnguyen In order to make the deletion work, we need to remove this save.
Is this going to create any other problem?

Reply:
@enricocolasante  I'm not sure, this code has been in the class AnalyticObjectBundleHook  for a long time before it was moved to DefaultAnalyticObjectImportHandle so that it can be reused in different places. 
I guess removing this save function would cause issues for the importer. Because DataElementOperand  is EmbeddedObject.
You can try and check if this test is ok `MetadataImportServiceTest.testImportEmbeddedObjectWithSkipSharingIsTrue`

Reply:
I updated the test and the import works correctly.
`DataElemenentOperand` is an embedded object but in in this case is not treated as one because of the hibernate mapping.